HW1/P3/P3.txt

```diff
@@ -1 +1 @@
-# Your discussion here
+[(u'aelst', (11, [u'least', u'leats', u'salet', u'slate', u'stale', u'steal', u'stela', u'taels', u'tales', u'teals', u'tesla']))]
```

Comment:
Interesting, it looks like there are multiple word combinations that work for this problem. My answer used a different combination of words.